data/sql/updates/pending_db_world/webwinder-signpost-gps.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE `gameobject` SET `position_x` = '634.220', `position_y` =  '359.989' WHERE (`guid` = '47468' and id = '19569');
+UPDATE `gameobject` SET `position_x` = '634.065', `position_y` =  '360.295' WHERE (`guid` = '47480' and id = '19570');
+UPDATE `gameobject` SET `position_x` = '633.923', `position_y` =  '359.858', `position_z` = '50.713' WHERE (`guid` = '47481' and id = '19571');
```

Comment:
```suggestion
UPDATE `gameobject` SET `position_x` = 633.923, `position_y` =  359.858, `position_z` = 50.713 WHERE `guid` = 47481 AND `id` = 19571;
```
and add whiteline at end. also same formatting for the others brackets don't really matter that much afaik but I left them out